app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/CafeteriaDao.java

```diff
@@ -15,6 +15,9 @@
     @Query("SELECT * FROM cafeteria")
     Flowable<List<Cafeteria>> getAll();
 
+    @Query("SELECT * FROM cafeteria")
+    List<Cafeteria> getAllNow();
```

Comment:
You could also consider using `blockingFirst()` on the Flowable instead of adding a dedicated `getAllNow()`.